specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json

```diff
@@ -0,0 +1,1282 @@
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The detail status of individual documents",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DocumentStatusDetail"
+          }
+        },
+        "@nextLink": {
+          "description": "Url for the next page.  Null if no more pages available",
+          "type": "string",
+          "example": "https://westus.cognitiveservices.azure.com/translator/text/batch/v1.0.preview.1/operations/727BF148-F327-47A0-9481-ABAE6362F11E/documents?$top=5&$skip=15"
+        }
+      }
+    },
+    "FileFormat": {
+      "type": "object",
+      "properties": {
+        "format": {
+          "description": "Name of the format",
+          "type": "string",
+          "example": "PlainText"
+        },
+        "fileExtensions": {
+          "description": "Supported file extension for this format",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "contentTypes": {
+          "description": "Supported Content-Types for this format",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "versions": {
+          "description": "Supported Version",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "FileFormatListResult": {
+      "description": "Base type for List return in our api",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "list of objects",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FileFormat"
+          }
+        }
+      }
+    },
+    "StorageSourceListResult": {
+      "description": "Base type for List return in our api",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "list of objects",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StorageSource"
+          }
+        }
+      }
+    },
+    "ErrorResponseV2": {
+      "description": "Contains unified error information used for HTTP responses across any Cognitive Service. Instances\r\ncan be created either through Microsoft.CloudAI.Containers.HttpStatusExceptionV2 or by returning it directly from\r\na controller.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorV2"
+        }
+      }
+    }
+  },
+  "securityDefinitions": {
+    "subscription_key": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header",
+      "description": "Provide your cognitive services subscription key here."
+    }
+  },
+  "security": [
+    {
+      "subscription_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}/translator/text/document/v1.0-preview.1",
```

Comment:
both `hostTemplate` and `path` have `v1.0-preview.1`. Looks suspecious.

Reply:
thanks.  this is fixed now